app/gui/view/component-browser/component-list-panel/breadcrumbs/src/entry.rs

```diff
@@ -92,6 +95,10 @@ pub mod ellipsis {
 // === Style ===
 // =============
 
+/// The width of the icon in a text [`Entry`].
+pub const ICON_WIDTH: f32 = 30.0;
```

Comment:
On the attached screenshot, I see the width of the gap between `data ▶︎` and `icon read` to be enormous. I wonder if it is caused by incorrect size or the fact that we don't have separate padding and size settings for the icon. Also, shouldn't it be in the stylesheet?

Reply:
Fixed the gap and this is now matching the design. This was caused by too large icon size, resulting in space around the icon.

> Also, shouldn't it be in the stylesheet?
All the icon sizes here are fixed, so this is following the current implementation.